taskcluster/config.yml

```diff
@@ -18,39 +18,40 @@ taskgraph:
 # The list of valid stages that can be used with `target-stage and `start-stage`.
 # These get attached to tasks in `kinds`.
 valid-stages:
-    - clean-corpus
+    - clean-parallel
     - clean-mono
     - bicleaner
     - bicleaner-model
-    - merge-corpus
+    - merge-cleaned-parallel
     - merge-devset
-    - merge-mono
+    - merge-cleaned-mono
     - train-vocab
     - train-backwards
     - evaluate-backwards
-    - split-corpus
-    - split-mono
-    - translate-mono-trg
-    - collect-mono-trg
+    - split-distillation-parallel-src
+    - split-backtranslations
+    - split-distillation
+    - translate-backtranslations-mono-trg
+    - collect-distillation-mono
     - train-teacher
     - evaluate-teacher
     - evaluate-finetuned-teacher
-    - translate-corpus
-    - extract-best
-    - collect-corpus
-    - translate-mono-src
-    - collect-mono-src
-    - merge-translated
-    - score
-    - cefilter
+    - translate-distillation-parallel-src
+    - distillation-parallel-extract-best
+    - collect-distillation-parallel
+    - translate-distillation-mono-src
+    - collect-backtranslations
+    - merge-distillation
+    - distillation-corpus-score
+    - distillation-corpus-keep-best
     - alignments-teacher
```

Comment:
There should be separate stages for "alignments-original" and "alignments-backtranslated". I think I ran into an  issue of having one stage for two steps a couple of times.

Reply:
or maybe it should be called "alignments-parallel" and "alignments-backtranslations" following your new naming scheme